charts/kafka/values.yaml

```diff
@@ -42,8 +42,11 @@ resources: {}
   #   memory: 1024Mi
 kafkaHeapOptions: "-Xmx1G -Xms1G"
 
-## Optional Container Security context
-securityContext: {}
+## Kafka pod Security Context
+securityContext:
+  enabled: true
+  fsGroup: 0
+  runAsUser: 0
```

Comment:
Am not keen on using root here but we can tackle this later when it becomes an issue running this on OpenShift